README.md

```diff
@@ -133,3 +133,60 @@ var intValue: Int
 var intValue : Int
 
 ```
+
+### Enums ###
+
+For enum declarations, declare each enum case on a new line with its own `case` statement instead of a comma-separated list.
+
+```swift
+internal enum State
```

Comment:
I'd say it would be good to specify it, that shows you are conscious of it. But maybe it's overkill? Or it would be overkill to make a rule of it?